include/Track.h

```diff
@@ -192,7 +192,12 @@ class TrackContentObjectView : public selectableObject, public ModelView
 
 // theming qproperties
 	Q_PROPERTY( QColor fgColor READ fgColor WRITE setFgColor )
+	Q_PROPERTY( QColor fgMutedColor READ fgMutedColor WRITE setFgMutedColor )
+	Q_PROPERTY( QColor mutedColor READ mutedColor WRITE setMutedColor )
+	Q_PROPERTY( QColor selectedColor READ selectedColor WRITE setSelectedColor )
 	Q_PROPERTY( QColor textColor READ textColor WRITE setTextColor )
+	Q_PROPERTY( QColor textShadowColor READ textShadowColor WRITE setTextShadowColor )
+	Q_PROPERTY( bool gradient READ gradient WRITE setGradient )
```

Comment:
I think I remember discussing this so sorry if I've switched gears, but is a `bool` necessary here?  Can't we simply set the alpha channel to `0` on the necessary colors?